Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.yml

```diff
@@ -186,7 +239,537 @@ script:
     - contextPath: RecordedFuture.Credentials.Identities.credentials.exposed_secret.details.rank
       description: Rank for the exposed password.
       type: String
-  runonce: false
+  - name: recordedfuture-identity-playbook-alerts-search
+    description: Search playbook alerts based on filters.
+    arguments:
+    - name: limit
```

Comment:
Maybe you'll want to add a default value here?

Reply:
yes, thank you, done